pytensor/graph/basic.py

```diff
@@ -907,6 +907,35 @@ def graph_inputs(
     yield from (r for r in ancestors(graphs, blockers) if r.owner is None)
 
 
+def required_graph_inputs(graph):
+    """
+    Get the inputs into PyTensor variables
```

Comment:
Docs need some work. They are not clear as to what's the purpose and signature seems off.

I suggest adding a minimal code example of how the function can be used